common/config/azure-pipelines/valgrind.yaml

```diff
@@ -0,0 +1,85 @@
+# iTwin.js Valgrind Testing
+
+trigger:
+  branches:
+    include:
+      - master
+
+jobs:
+  - job:
+    timeoutInMinutes: 180
+    pool:
+      name: iModelTechCI
```

Comment:
Can we use a standard hosted agent instead of this pool?

Reply:
Went back to pool because the hosted agent froze after 6 hours every time.